### ISSUE TITLE
lg() not expanding

### ISSUE THREAD

**Issue by ixarls** [emmetio/emmet#670]
linear gradient in .css not expanding.
I'm using VS code. lg works -> background-image: linear-gradient();
but lg(...)  does not. A have actually tried another editor a got the same result.
Am i doing something wrong?

**Comment by sergeche** [comment]
Looks like a bug, will check it out

**Comment by twofivetwo** [comment]
issue is still present.

**Comment by theperfectpunk** [comment]
@sergeche the following test cases pass in our build
```javascript
it('gradient resolver', () => {
    equal(expand('lg'), 'background-image: linear-gradient(${0});');
    equal(expand('lg(to right, #0, #f00.5)'), 'background-image: linear-gradient(to right, #000, rgba(255, 0, 0, 0.5));');
});
```
The issue can still be reproduced in vscode, what I've observed is emmet suggestions stop after a round bracket.
Maybe its an issue with implementation in the plugins

![lgexpand](https://user-images.githubusercontent.com/7138111/226172847-eee5145d-f731-487d-a256-32ec498348ad.gif)

**Comment by sergeche** [comment]
Yes, it’s VSCode implementation issue. Works fine in Sublime Text and CodeMirror.
You should fill an issue in VSCode repe
